### DISCUSSION TITLE
Copilot constantly 'loading'

### DISCUSSION THREAD

**Discussion by mattjohnpowell** [community/community#40623] [Copilot Conversations]
### Select Topic Area

Bug

### Body

It's been a while but Copilot really isn't working. The lower right icon is constantly showing a spinner, can anyone help me fix?

and this is from the Output:

[INFO] [ghostText] [2022-12-02T13:55:26.290Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:26.290Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:26.291Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:26.291Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:26.291Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:28.997Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:28.998Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:28.998Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:28.998Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:28.998Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:29.000Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.573Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.573Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.573Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.573Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.573Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.573Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.573Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.574Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.574Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.574Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.574Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.574Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.968Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T13:55:32.968Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:13.919Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:13.922Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:14.079Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:14.079Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:42.111Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:42.111Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:42.112Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:42.113Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-12-02T14:01:42.114Z] Cancelled after extractPrompt

**Comment by ossanna16** [comment]
Hi there @mattjohnpowell and welcome to our community! Thank you for reporting this issue!

**Comment by mattjohnpowell** [comment]
I think I'm going to have to cancel my subscription, it just becomes painful to vscode as there is now no longer an autocomplete and no Copilot
